<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminwindow</class>
 <widget class="QWidget" name="adminwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>611</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="listado">
    <property name="contextMenuPolicy">
     <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
    </property>
    <attribute name="title">
     <string>Listado</string>
    </attribute>
    <widget class="QLineEdit" name="cedulaEdit">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="placeholderText">
      <string>Ingrese la cédula para filtrar paquete por cliente...</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="packagesTable">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>38</y>
       <width>761</width>
       <height>541</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>7</number>
     </property>
     <column>
      <property name="text">
       <string>Cedula</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cliente</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Paquete</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Estado</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nombre del Producto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cantidad de Producto</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="loadButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="buscarCliente">
    <attribute name="title">
     <string>Buscar Cliente</string>
    </attribute>
    <widget class="QLineEdit" name="filterEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Ingrese la cédula del Cliente a buscar...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filterButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Filtrar</string>
     </property>
    </widget>
    <widget class="QLabel" name="filterLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>351</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="buscarID">
    <attribute name="title">
     <string>Buscar ID</string>
    </attribute>
    <widget class="QLineEdit" name="packageEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Ingrese el ID de paquete a buscar...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
    </widget>
    <widget class="QLabel" name="resultLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>291</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gestion">
    <attribute name="title">
     <string>Gestion</string>
    </attribute>
    <widget class="QLineEdit" name="manageEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>ID del paquete...</string>
     </property>
    </widget>
    <widget class="QComboBox" name="statusComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="placeholderText">
      <string>Ingrese una opción...</string>
     </property>
     <item>
      <property name="text">
       <string>En tránsito</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Entregado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cancelado</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="updateStatusButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualizar estado</string>
     </property>
    </widget>
    <widget class="QLabel" name="manageMessageLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="eliminar">
    <attribute name="title">
     <string>Eliminar paquete</string>
    </attribute>
    <widget class="QLineEdit" name="eliminarEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Ingrese el ID del paquete a eliminar...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eliminarButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="solicitudesTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>621</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="eliminarLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>24</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
